src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,80 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final int FIRST_PART = 0;
+    private static final int SECOND_PART = 1;
```

Comment:
contants should have informative names 
```suggestion
    private static final int OPERATION_TYPE_INDEX = 0;
    private static final int AMOUNT_INDEX = 1;
```